frontend/src/routes/(authenticated)/admin/UserFilter.svelte

```diff
@@ -0,0 +1,67 @@
+<script  context="module" lang="ts">
+  import type { User } from './+page';
+
+  export type UserFilter = {
+    userSearch: string;
+    usersICreated: boolean;
+  }
+
+  export function filterUsers(
+    users: User[],
+    userFilter: Partial<UserFilter>,
+    adminId: string | undefined,
+  ): User[] {
+    return users.filter(
+      (u) =>
+        (!userFilter.usersICreated || u.createdById === adminId)
+    );
+  }
+</script>
+
+<script lang="ts">
+  import type { Writable } from 'svelte/store';
+  import FilterBar from '$lib/components/FilterBar/FilterBar.svelte';
+  import ActiveFilter from '$lib/components/FilterBar/ActiveFilter.svelte';
+  import { Icon } from '$lib/icons';
+  import t from '$lib/i18n';
+
+
+  type Filters = Partial<UserFilter> & Pick<UserFilter, 'userSearch'>;
+  export let filters: Writable<Filters>;
+  export let filterDefaults: Filters;
+  export let hasActiveFilter: boolean = false;
+  export let autofocus: true | undefined = undefined;
+  export let filterKeys: (keyof Filters)[] = ['userSearch', 'usersICreated'];
+  export let loading = false;
+
+  function filterEnabled(filter: keyof Filters): boolean {
+    return filterKeys.includes(filter);
+  }
+</script>
+
+<FilterBar on:change searchKey="userSearch" {autofocus} {filters} {filterDefaults} bind:hasActiveFilter {filterKeys} {loading}>
```

Comment:
When we pass lots of attributes to a component, we typically put each one on a separate line, similar to how the UserFilter component is invoked on the admin dashboard page. There's no hard-and-fast rule for when to switch from all-in-one-line to one-per-line, but typically if there are more than four or five attributes, or if any of them involve a callback function, then we'll write them one per line.

Here, that would have made it slightly more obvious that the `debounce` attribute was missing; because these attributes are passed all in one line, it's easier for the eye to glaze over and miss the absence of one single attribute.